README.md

```diff
@@ -84,6 +85,21 @@ Please see [our Showcase page](https://weareadaptive.com/showcase/) for a full l
 
 We practice continuous integration and deployment. Every branch and pull request triggers a build and deployment to an ephemeral environment. Merging to master causes a build and deployment to our [development environment](https://web.dev.reactivetrader.com).
 
+## Reactive Launcher
```

Comment:
I think you should mention that with the Reactive Launcher, you can launch all other Reactive application such as RTC, RTA, LimitChecker etc..